contracts/LifToken.sol

```diff
@@ -457,14 +469,20 @@ contract LifToken is Ownable, ERC20, SafeMath, PullPayment {
         giveVotes(msg.sender, to);
       }
 
-      TransferData(msg.sender, to, value, data);
+      if (doCall && to.call(data))
+        TransferData(msg.sender, to, value, data);
+      else if (!doCall)
```

Comment:
We expect the call to always return a bool, so if doCall is false it will just jump to the next condition, where it will just log the event